tee-worker/litentry/core/credentials/src/templates/credential.json

```diff
@@ -4,6 +4,7 @@
         "https://w3id.org/security/suites/ed25519-2020/v1"
     ],
     "id":"",
+    "jsonschema": "",
```

Comment:
I notice there's a `credentialSchema` field in this json, are they duplicate?

Reply:
ah! my bad– I followed the ticket without checking the w3c standard. `credentialSchema` is suggested. We added that long ago but never made use of it. I'll use it better. Thanks, Kai!